packages/shared_preferences/shared_preferences_android/android/build.gradle

```diff
@@ -2,7 +2,7 @@ group 'io.flutter.plugins.sharedpreferences'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.7.10'
+    ext.kotlin_version = '1.9.10'
```

Comment:
@reidbaker I matched this to the kotlin version in other plugins.  The tests passed, are there other implications to bumping this?  I didn't see anything when I looked at other PRs, like #6228 (also a minor pubspec version bump)